tests/units/anta_tests/test_interfaces.py

```diff
@@ -2441,4 +2442,135 @@
             ],
         },
     },
+    {
+        "name": "success",
+        "test": VerifyLACPInterfacesStatus,
+        "eos_data": [
+            {
+                "portChannels": {
+                    "Port-Channel5": {
+                        "interfaces": {
+                            "Ethernet5": {
+                                "actorPortStatus": "bundled",
+                                "partnerPortState": {
+                                    "activity": True,
+                                    "timeout": False,
+                                    "aggregation": True,
+                                    "synchronization": True,
+                                    "collecting": True,
+                                    "distributing": True,
+                                    "defaulted": False,
+                                    "expired": False,
```

Comment:
I think we should (again) update the docstring to mention these parameters to let the user know what exactly we are checking.

Reply:
Updated the docstring. Thanks!!